_pages/openid-connect.md

```diff
@@ -259,18 +261,63 @@ grant_type=authorization_code
 
     ```json
     {
+      "sub": "b2d2d115-1d7e-4579-b9d6-f8e84f4f56ca",
       "iss": "https://idp.int.login.gov",
+      "acr": "http://idmanagement.gov/ns/assurance/loa/1",
+      "nonce": "aad0aa969c156b2dfa685f885fac7083",
       "aud": "urn:gov:gsa:openidconnect:development",
-      "sub": "9e49553d-894b-4e53-917a-297824fdf96f",
-      "acr": "http://idmanagement.gov/ns/assurance/loa/3",
-      "nonce": null,
-      "jti": "c0b8d9386483327342727e7d750bc6a1",
-      "exp": 1486478556,
-      "iat": 1486478558,
-      "nbf": 1486478558
+      "jti": "jC7NnU8dNNV5lisQBm1jtA",
+      "at_hash": "tlNbiqr1Lr2YcNRGjzwlIg",
+      "c_hash": "hXjq7kOrtQK_za_6tONxcw",
+      "exp": 1489694196,
+      "iat": 1489694198,
+      "nbf": 1489694198
     }
     ```
 
+    <div class="usa-accordion">
+    <button class="usa-accordion-button" aria-controls="id-token-details">
+    View JWT claim details
+    </button>
+    <div id="id-token-details" class="usa-accordion-content" markdown="1">
+    The decoded `id_token` contains a few claims:
+
+    * **acr**
+      Authentication Context Class Reference value or LOA (level of authentication) of the returned claims, from the original [authorization request](#authorization-request).
+
+    * **at_hash**
+      Access token hash, a url-safe base-64 encoding of the left 128 bits of the SHA256 of the `access_token` value. Provided so the client can verify the `access_token` value.
+
+    * **aud**
+      Audience, the client ID.
+
+    * **c_hash**
+      Code hash, a url-safe base-64 encoding of the left 128 bits of the SHA256 of the authorization `code` value. Provided so the client verify the `code` value.
+
+    * **exp**
+      Expiration, an integer timestamp of the expiration of this token (number of seconds since the Unix Epoch).
+
+    * **iat**
+      Issued at, an integer timestamp of when the token was created (number of seconds since the Unix Epoch).
+
+    * **iss**
+      Issuer, will be `https://idp.int.login.gov`.
+
+    * **jti**
+      An random string generated to ensure uniqueness.
+
+    * **nbf**
+      "Not before", an integer timestamp of when the token will start to be valid (number of seconds since the Unix Epoch).
+
+    * **nonce**
+      The nonce provided by the client in the [authorization request](#authorization-request)
```

Comment:
should we clarify that this is a string? It seems like in most of these field descriptions you include the TYPE